docs/3.tutorials/examples/blockvote-js-edition.md

```diff
@@ -0,0 +1,323 @@
+# BlockVote JS Edition
+
+BlockVote JS Edition is a blockchain-based voting application built using JavaScript on the Near Protocol blockchain. The application allows users to securely cast their votes in an election and have them recorded on the blockchain, ensuring that the results are transparent and cannot be altered.
+
+<p align="center"><img src="https://i.imgur.com/spJWhBf.png"></p>
```

Comment:
Let's try to upload any images or resources to our repo to avoid issues if `imgur.com` goes down or something like that.

You can copy the image to this path in the repo:
```
/website/static/docs/assets/
```

and then link to `/docs/assets/my-file.png` in the markdown file